lib/repo/git_repository.rb

```diff
@@ -489,7 +597,8 @@ def remove_user(user_id)
           raise UserNotFound.new(user_id + " not found")
         end
       else
-        raise NotAuthorityError.new("Unable to modify permissions: Not in authoritative mode!")
+        raise NotAuthorityError.new(
+                  "Unable to modify permissions: Not in authoritative mode!")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.